docs/deprecations.md

```diff
@@ -74,24 +90,32 @@ Eventually if you have a strong reason to stick to current convention, you may e
 
 ## Support for `alexaSkill` event without `appId` is to be removed
 
+Deprecation code: `ALEXA_SKILL_EVENT_WITHOUT_APP_ID`
+
 Starting with v3.0.0, support for `alexaSkill` event without `appId` provided will be removed.
 
 <a name="AWS_KMS_KEY_ARN"><div>&nbsp;</div></a>
 
 ## `awsKmsKeyArn` references
 
+Deprecation code: `AWS_KMS_KEY_ARN`
+
 Please use `provider.kmsKeyArn` and `functions[].kmsKeyArn`. `service.awsKmsKeyArn` and `functions[].awsKmsKeyArn` will be removed with v3.0.0
 
 <a name="RESOURCES_EXTENSIONS_REFERENCE_TO_NONEXISTENT_RESOURCE"><div>&nbsp;</div></a>
 
 ## Defining extensions to nonexistent resources in `resources.extensions`
 
+Deprecation code: `RESOURCES_EXTENSIONS_REFERENCE_TO_NONEXISTENT_RESOURCE`
+
 Starting with v3.0.0, extensions to nonexistent resources in `resources.extensions` will throw an error instead of passing silently.
 
 <a name="DISABLE_LOCAL_INSTALLATION_FALLBACK_SETTING"><div>&nbsp;</div></a>
 
 ## Support for `enableLocalInstallationFallback` setting is to be removed
 
+Deprecation code: `DISABLE_LOCAL_INSTALLATION_FALLBACK_SETTING`
```

Comment:
Let's also add these codes below, for deprecations that were removed with v2.0.0, as some people might still be on lower versions for some reason

Reply:
I have updated both the review comments.  Please check and let me know your feedback.  Thank you